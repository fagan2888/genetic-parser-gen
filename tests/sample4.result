((lp0
cprimitives
str_remove_tabs
p1
ag1
ag1
ag1
acprimitives
str_remove_consecutive_spaces
p2
a(lp3
(S'([0-9]{2}/[0-9]{2})'
p4
I1
tp5
a(S'\\s'
p6
I1
tp7
a(S'.{1,80}?'
p8
I0
tp9
ag7
ag9
ag7
a(S'(.{1,80}?)'
p10
I0
tp11
ag7
ag11
a(S'(-?[0-9]{1,20})'
p12
I1
tp13
a(S'(\n)'
p14
I1
tp15
ag9
a(S'(-?[0-9]{1,20}[.,]+[0-9]{1,20})'
p16
I1
tp17
ag7
a(S'[\'"\\-+,;:\\.]'
p18
I1
tp19
a(lp20
cprimitives
clean_merge5
p21
acprimitives
clean_strip
p22
ag21
ag22
acprimitives
clean_merge3
p23
atp24
.