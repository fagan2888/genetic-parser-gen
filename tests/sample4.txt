Date          : 12/12
Libellé       : 01230123
                01230123
AAAAA         : 200,00
+--------------------------------------------------------------------+
Date          : 12/12
Libellé       : TELECOM
                127538228
AAAAA         : -29,99
+--------------------------------------------------------------------+
