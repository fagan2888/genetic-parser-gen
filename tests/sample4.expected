"12/12","01230123","01230123","200,00"
"12/12","TELECOM","127538228","-29,99"
